modules/serverless-install-kafka-odc.adoc

```diff
@@ -0,0 +1,34 @@
+// Module is included in the following assemblies:
+//
+// serverless/serverless-kafka.adoc
+
+[id="serverless-install-kafka-odc_{context}"]
+= Installing Knative Kafka components
+
+Cluster administrators can enable the use of Kafka functionality in a {ServerlessProductName} deployment by instantiating the Kafka custom resource definition provided by the *Knative Kafka* {ServerlessOperatorName} API.
+
+.Prerequisites
+// * You must have access to an AMQ Streams Kafka cluster.
```

Comment:
I think `AMQ Streams Kafka` is a viloation of Apache trademarks. I'd say something like:

You must have access to an Apache Kafka cluster, we do recommend AMQ Streams.

NOTE: any Apache Kafka cluster should work (like strimzi or even the `wurstmeister` images)

Reply:
IMO it would be better to say just "You must have access to an AMQ Streams cluster". I know that others will work but from a product docs perspective we should reference the supported product. 

Also, on the trademark, the first use of Kafka on each page should be "Apache Kafka", after that just "Kafka" can be used. That's my understanding anyway.

Reply:
`AMQ Streams Kafka` -> sounds like a product name - but it's `AMQ Streams`. 

I just wanted to say that even naming a product like "Blah Blah Kafka" is illegal :-)